src/main/java/io/nats/client/impl/NatsJetStream.java

```diff
@@ -540,6 +540,7 @@ public List<String> getChanges(ConsumerConfiguration serverCc) {
             if (idleHeartbeat != null && !idleHeartbeat.equals(getOrUnset(serverCcc.idleHeartbeat))) { changes.add("idleHeartbeat"); }
             if (maxExpires != null && !maxExpires.equals(getOrUnset(serverCcc.maxExpires))) { changes.add("maxExpires"); }
             if (inactiveThreshold != null && !inactiveThreshold.equals(getOrUnset(serverCcc.inactiveThreshold))) { changes.add("inactiveThreshold"); }
+            if (priorityTimeout != null && !priorityTimeout.equals(getOrUnset(serverCcc.priorityTimeout))) { changes.add("priorityTimeout"); }
```

Comment:
This check comes into play when there is an old style sub where they always supply the consumer config.